docs/bigcommerce-for-wordpress/extending-the-plugin/proxy-rest-api-endpoints.mdx

```diff
@@ -14,15 +14,15 @@ By default, GET requests under `/catalog` are cached for ten minutes in the obje
 
 |URL|Description|Methods|Excluded Fields
 |:-|:-|:-|:-|

```

Comment:
for this whole table:

* remove backticks from links. this usually means redescribing links with words rather than the form of the endpoint. for example, replace 
```
[`/catalog/products(/*)`](/api-reference/catalog/products/getproducts)
```

with 

```
[Catalog products endpoints](correct link)
```

Reply:
the other thing is that the links are largely broken despite valiant effort, so check with @bc-stephen-hilliard on collaborating to get those updated.